server/src/test/java/com/broadcom/lsp/cobol/usecases/TestSkipStatementsAllowedEverywhere.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+
+package com.broadcom.lsp.cobol.usecases;
+
+import com.broadcom.lsp.cobol.service.CopybookProcessingMode;
+import com.broadcom.lsp.cobol.usecases.engine.UseCaseEngine;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This test checks that SKIP1, SKIP2 and SKIP3 are allowed everywhere and do not produce syntax
+ * errors.
+ */
+class TestSkipStatementsAllowedEverywhere {
+  private static final String TEXT =
+      "        IDENTIFICATION DIVISION.\n"
+          + "        SKIP1\n"
+          + "        PROGRAM-ID. TEST1.\n"
```

Comment:
Here I expect that line with `PROGRAM-ID` will be skipped. So that means that here will be an error on `DATA DIVISION` line: "Syntax error on 'DATA' expected PROGRAM_ID".

Reply:
The SKIP1, SKIP2, and SKIP3 statements specify blank lines that the compiler should add when printing the source listing. SKIP statements have no effect on the compilation of the source text itself.

Reply:
Sorry, my bad.